src/main/kotlin/org/rust/lang/core/psi/ext/RsDocAndAttributeOwner.kt

```diff
@@ -89,6 +92,8 @@ class QueryAttributes(
             val stub = psi.greenStub
             if (stub != null) return stub.isCfg
         }
+        // TODO: We probably want this optimization also for other items that we query regularly
```

Comment:
Seems reasonable:
![image](https://user-images.githubusercontent.com/3221931/63335685-7e8fef80-c346-11e9-8f9a-72b217237159.png)

Looks like these items are queried most often:
![image](https://user-images.githubusercontent.com/3221931/63335862-cca4f300-c346-11e9-8581-aaa018dcf5b8.png)

Also, let's add the same flag to `RsImplItem` stub, it will be used later in `RsImplIndex`, and `RsModDecl`